### PR TITLE
Provide context about how the extension was invoked

### PR DESCRIPTION
**Pull request by nmehta6**

### CODE REVIEW

File: commands/interactiveInfo.ts [hunk 1]
```diff
@@ -2,5 +2,28 @@ import {getDeviceType, deviceTypes} from './device'
...
 
 export default function getInteractiveInfo () {
+
+	const interactiveContext = JSON.parse(sessionStorage['viewerInteractiveContext'])
 	return get('interactive')
+		.then(info => {
+			if (!!interactiveContext) {
+
+				switch (interactiveContext.type) {
+					case 'collection':
```
Comment (nmehta6) [inline commands/interactiveInfo.ts]: There were other cases such as story mapper and search where we also preserve context, but that seemed unnecessary at this point. So, I have only added support for folders and collections.

File: commands/interactiveInfo.ts [hunk 2]
```diff
@@ -2,5 +2,28 @@ import {getDeviceType, deviceTypes} from './device'
 import { get } from './internalMethods'
 
 export default function getInteractiveInfo () {
+
+	const interactiveContext = JSON.parse(sessionStorage['viewerInteractiveContext'])
```
Comment (nmehta6) [inline commands/interactiveInfo.ts]: This is how Viewer preserves context for Collections, search, etc. There is Viewer frontend counterpart of this.

Comment (eugirdor) [reply commands/interactiveInfo.ts]: How does this work for other platforms?  Assuming they just have to return the info in the `interactive` call?

Comment (nmehta6) [reply commands/interactiveInfo.ts]: Yeah that's right. These values are stored in the sessionStorage by Viewer only.

**Pull request merged**